src/App.js

```diff
@@ -25,11 +26,21 @@ import useDefaultDragAndDrop from './hooks/useDefaultDragAndDrop';
 import OnyxUpdateManager from './libs/actions/OnyxUpdateManager';
 import * as Session from './libs/actions/Session';
 import * as Environment from './libs/Environment/Environment';
+import InitialUrlContext from './libs/InitialUrlContext';
```

Comment:
Is there a reason why those changes are here and not e.g. in `Expensify` file?

Reply:
That's because we use initial props, that are passed directly to `App.js` file. 😄